Extensions/Content/Dnn.PersonaBar.Pages/Components/Prompt/Commands/DeletePage.cs

```diff
@@ -55,7 +55,7 @@ public override ConsoleResultModel Run()
             }
             try
             {
-                PagesController.Instance.DeletePage(new PageItem { Id = PageId }, PortalSettings);
+                PagesController.Instance.DeletePage(new PageItem { Id = PageId }, false, PortalSettings);
```

Comment:
By the way why add the new parameter in the middle? If it was added to the end of the signature then this usage would not have to have been touched.